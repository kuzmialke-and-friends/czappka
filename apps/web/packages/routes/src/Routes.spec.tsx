import React from 'react';
import { render } from '@testing-library/react';
import { Routes } from './Routes';

describe('Routes', () => {
    it('renders correctly', () => {
        const { container } = render(<Routes />);

        expect(container).toMatchSnapshot();
    });
});
