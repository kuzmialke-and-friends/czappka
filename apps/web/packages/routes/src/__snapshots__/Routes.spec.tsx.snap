// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes renders correctly 1`] = `
<div>
  <h1>
    test
  </h1>
</div>
`;
